pkg/cloudprovider/aws/aws.go

```diff
@@ -1710,6 +1733,31 @@ func (s *AWSCloud) ensureSecurityGroup(name string, description string, vpcID st
 	return groupID, nil
 }
 
+// createTags calls EC2 CreateTags, but adds retry-on-failure logic
+// We retry mainly because if we create an object, we cannot tag it until it is "fully created" (eventual consistency)
+// The error code varies though (depending on what we are tagging), so we simply retry on all errors
+func (s *AWSCloud) createTags(request *ec2.CreateTagsInput) (*ec2.CreateTagsOutput, error) {
+	attempt := 0
```

Comment:
I really should be doing exponential backoff here I think.  AWS had a pretty bad outage in the early days because everyone was just doing straight loops, and so a degraded API grew into a full on outage, as I recall.  I hope we can implement that using a reusable Go function but I'm not entirely sure how best to express that.  Maybe something like time.Tick but with a timeout and backoff?   I'll add a TODO to add exponential backoff in the meantime.